src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -137,22 +148,36 @@ public class PullRequestHandler : IHttpHandler
         return RunGitCommands(gitCommands, executionResult);
     }
 
+    private bool SyncToBranch(string branch, StringBuilder executionResult)
+    {
+        string[] gitCommands = new string[]
+        {
+            "clean -fdx",
+            "checkout master",
```

Comment:
Since you asked people to pass in `branch` as a parameter to the function, please use it :smile:.